test/python/test_rticonnextdds_input.py

```diff
@@ -60,3 +61,24 @@ def test_reader_native_call(self, rtiInputFixture):
     get_name.restype = ctypes.c_char_p
     get_name.argtypes = [ctypes.c_void_p]
     assert rti.fromcstring(get_name(topic)) == "Square"
+
+  def test_no_autoenable(self):
+    with open_test_connector("MyParticipantLibrary::TestNoAutoenableSubscriber") as connector:
+      output = connector.get_output("TestPublisher::TestWriter")
+      connector.wait(1000) # TODO
+
+      # The input is not enabled yet because the subscriber sets
+      # autoenable_created_entities to false
+      output.instance['x'] = 1
+      output.write()
+
```

Comment:
You could call connector.wait() and check that a timeout error is raised

Reply:
Checking that only the second sample is received is enough, and doesn't add an additional sleep time.